### DISCUSSION TITLE
Cannot use Inception_v3 with Avalanche

### DISCUSSION THREAD

**Discussion by caesar753** [ContinualAI/avalanche#1289] [Q&A] [answered]
Hello,

I'm trying to use Avalanche with  pretrained CNNs, downloading them from torch.hub

`model = torch.hub.load('pytorch/vision:v0.10.0', "inception_v3", pretrained=True, force_reload=True)`

it works well with various kind of ResNet (18, 32, 50, ...), but when I use Google Inception_v3 it gives to me this error

`TypeError: cross_entropy_loss(): argument 'input' (position 1) must be Tensor, not InceptionOutputs`

**Comment by HamedHemati** [comment, marked as answer]
The output of InceptionV3 is a tuple by default, and Avalanche expects the model's output to be logits. You can disable the auxiliary output in InceptionV3 by setting `aux_logits=False` when initializing your model:

```
model = torch.hub.load('pytorch/vision:v0.10.0', "inception_v3", pretrained=True, force_reload=True, aux_logits=False)
```

**Reply by caesar753** [reply]
Great, it works!
